### PR TITLE
remove tooltip v2 payment modal sticker

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?
removes the tooltip in v2 to match v1 functionality addressed down the pr stack

hover sticker icon, tool tip doesn't show anymore 
## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1297** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1297" target="_blank">Graphite</a> 
    * **PR #1506** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1506" target="_blank">Graphite</a> 
      * **PR #1508** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1508" target="_blank">Graphite</a> 
        * **PR #1509** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1509" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1509?utm_source=stack-comment).

**Review by tomquirk — APPROVED** [review]

**Comment by tomquirk** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1509?utm_source=stack-comment).


Job Reference: FBdSlkLHRNUzrDzyrjTF

**Comment by tomquirk** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1509?utm_source=stack-comment).


Job Reference: hcVmZSl0NtpcQYTG1Iw4

**Comment by tomquirk** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1509?utm_source=stack-comment).


Job Reference: ukIh7TJlk56j5lLaSiwO

**Comment by tomquirk** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1509?utm_source=stack-comment).


Job Reference: jrBSRizfw0H0WEnCyQfl

**Pull request merged**
